### PR TITLE
Api endpoints update

### PR DESCRIPTION
**Pull request by jamesaimonetti**

### CODE REVIEW

File: applications/crossbar/doc/ref/tasks.md [hunk 1]
```diff
@@ -4,9 +4,6 @@
 
 #### Schema
 
-Key | Description | Type | Default | Required
---- | ----------- | ---- | ------- | --------
-`records` | List the rows of input data | `array(object)` |   | `true`
```
Comment (fenollp) [inline applications/crossbar/doc/ref/tasks.md]: Uh you sure?

Comment (jamesaimonetti) [inline applications/crossbar/doc/ref/tasks.md]: Uh, is there a schema file?

Comment (fenollp) [inline applications/crossbar/doc/ref/tasks.md]: Yes but it's called tasks.records. Should I rename it to tasks?

Comment (jamesaimonetti) [inline applications/crossbar/doc/ref/tasks.md]: If you have a custom schema that doesn't follow the convention, it should go direct into the `doc/tasks.md` version and not the auto-generated version.

**Pull request merged**